test/data.test.js

```diff
@@ -0,0 +1,200 @@
+const request = require('supertest');
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+
+const app = require('../app');
+const expect = chai.expect;
+
+chai.use(chaiAsPromised);
+
+describe("functional -data ", () => {
```

Comment:
This file is pretty long just for a single test. Can this be organised in a better way ?